### PR TITLE
Mission

### PR DESCRIPTION
**Pull request by nelsonic**
+ [x] Attempting to answer the question: What is @dwyl's _Mission_?

Read the file: https://github.com/dwyl/start-here/blob/mission/mission.md

### CODE REVIEW

**Comment by nelsonic** [comment]
@iteles please take a look at this doc and if it's "OK", please merge. 
The focal part is: https://github.com/dwyl/start-here/blob/mission/mission.md#what-is-dwyls-mission

If it's OK as a "first pass", please merge.
I _very much_ encourage you to refine it if you have time (_either now or in the future_).

**Comment by nelsonic** [comment]
I'm linking to this from another doc ... would be good to get it reviewed soon (_tomorrow morning is fine_).

File: mission.md [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+
+## Context
+
+We have always had a
+["_Why_?"](https://github.com/dwyl/start-here#why)
+goal (_or several goals depending on who you ask_) for @dwyl.
+
+_Long_ before we started @dwyl we had been working for charities
+and non-profit organisations on a variety of missions
+including poverty reduction, cancer research, homelessness,
```
Comment (iteles) [inline mission.md]: I'll submit a PR to add environmental concerns to this section 🙌

File: mission.md [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+fair trade and education.
+
+One of the things that became clear to us
+while we were working for various charities
+and for-profit businesses
+was that in _most_ cases time was not used effectively
+and communication channels were poor.
+Even though mission statements were clear,
+_translating_ those words into the day-to-day actions
```
Comment (iteles) [inline mission.md]: ❤️

File: mission.md [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+If you more find good examples of mission statements,
+please add them!
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat-square)](https://github.com/dwyl/start-here/issues)
+
+<hr />
+
+
+# What is _@dwyl's_ `Mission`?
+
+### "_Empower people to maximise their personal effectiveness, creativity and happiness._"
```
Comment (iteles) [inline mission.md]: ❤️  I would be tempted to add 'whilst reducing their environmental impact' but I don't want to make the mission:
a) murky
b) appear to only appeal to the Western world/people of certain means (which traditionally caring for the environment has been)

We have at various points also discussed the possibility of adding 'through the use of technology' or words to that effect. I am motivated by a technology-driven organisation and that's where my heart lies but the mission will evolve as we ask questions of it in the short term.

Maybe we can revise this to add it in at a later date, definitely not needed for a starting point.

File: mission.md [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+
+Our App and Community should help people
+to discover meaningful creative work.
```
Comment (iteles) [inline mission.md]: My first impression here is that this sentence makes us sound a bit like a job board. I'll have a deeper think about this tonight and make suggestions, as per your #helpwanted comment below ☺️

File: mission.md [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+We should proactively share all our learning and knowledge
+and encourage everyone else to do the same
+so that we can accelerate progress
+and build a better future together.
```
Comment (iteles) [inline mission.md]: 🙌  Improve yourself, work with others and the world will be better as a default.
Start small.

File: mission.md (hunk 3 above)
Comment (nelsonic) [reply mission.md]: @iteles indeed we have discussed the wording "_through technology_" in the past.
The _reason_ I've left it out (_this time_) is:
a) to keep it succinct 📏 
b) often doing what you love will involve _escaping_ from tech 📵 

We could even remove creativity and happiness and _just_ focus on ***personal effectiveness***,
but I feel that removing the "joy" from our mission would make us robots and nobody wants that. 😉

File: mission.md (hunk 1 above)
Comment (nelsonic) [reply mission.md]: Yes, please do. 👍

File: mission.md (hunk 4 above)
Comment (nelsonic) [reply mission.md]: @iteles indeed. we _definitely_ need help with improving it. 👍 
but I was weary of _not_ having a clearly defined mission for so long ...  😩

**Comment by nelsonic** [comment]
@iteles thanks for reviewing & merging. :shipit: 
100% agree that further iteration/improvement are _essential_. 📝 
I look forward to readying _your_ PR(s) on this in the near future. ❤️ ✅

**Pull request merged**